src/proj.rs

```diff
@@ -1018,7 +1018,9 @@ impl convert::TryFrom<(&str, &str)> for Proj {
     }
 }
 
-struct PjInfo {
+/// Info about the current PROJ definition
```

Comment:
What do you think about linking to the reference docs for more context?

```suggestion
/// Info about the current PROJ definition
///
/// [PROJ reference documentation](https://proj.org/development/reference/datatypes.html?highlight=has_inverse#c.PJ_PROJ_INFO)
```